plugins/scaffolder-backend/src/scaffolder/dryrun/createDryRunner.ts

```diff
@@ -94,18 +91,23 @@ export function createDryRunner(options: TemplateTesterCreateOptions) {
       ]),
     });
 
-    const dryRunId = uuid();
+    // Extracting contentsPath and dryRunId from the baseUrl
+    const baseUrl = input.templateInfo.baseUrl;
+    if (!baseUrl) {
+      throw new Error('baseUrl is required');
+    }
+    const basePath = fileURLToPath(new URL(baseUrl));
+    const contentsPath = basePath.replace('template.yaml', '');
+    const dryRunId = contentsPath
```

Comment:
I think it should be fine to generate a new `dryRunId` in here instead of parsing it from the `basePath`

Reply:
I went ahead and did this and as expected you are on the money. All tests passed and my simple template that just wrote a log message and fetched one py file still worked. I was a little surprised because I thought the `getWorkspaceName` function below would break if the `dryrunId` generated here and the one used in basePath didn't match.